bridge_adaptivity/module/tests/test_grading_policy.py

```diff
@@ -105,3 +110,84 @@ def test_policy_math(
             grade = GradingPolicyCls(sequence=sequence, policy=policy)._calculate()
             self.assertEqual(grade, er)
             self.assertIsInstance(grade, (float, int))
+
+
+class TestPolicySendGradeMethod(TestCase):
+
+    fixtures = ['gradingpolicy.json', 'engine.json']
+
+    @mock.patch('module.tasks.sync_collection_engines.apply_async')
+    def setUp(self, mock_apply_async):
+        self.rf = RequestFactory()
+        self.user = BridgeUser.objects.create_user(
+            username='test_user',
+            password='test_pass',
+            email='test@test.com'
+        )
+        self.collection = Collection.objects.create(name='testcol1', owner=self.user)
+        self.collection2 = Collection.objects.create(name='testcol2', owner=self.user)
+
+        self.source_launch_url = 'http://test_source_launch_url.com'
+        self.activity = Activity.objects.create(
+            name='testactivity1', collection=self.collection, source_launch_url=self.source_launch_url
+        )
+        self.activity2 = Activity.objects.create(
+            name='testactivity2', collection=self.collection2, source_launch_url=self.source_launch_url
+        )
+        self.lti_provider = LtiProvider.objects.create(
+            consumer_name='test_consumer', consumer_key='test_consumer_key', consumer_secret='test_consumer_secret'
+        )
+        self.lti_user = LtiUser.objects.create(
+            user_id='test_user_id', lti_consumer=self.lti_provider, bridge_user=self.user
+        )
+        self.engine = Engine.objects.get(engine='engine_mock')
+        self.grading_policy = GradingPolicy.objects.get(name='trials_count')
+        self.outcome_service = OutcomeService.objects.create(
+            lis_outcome_service_url='test_url', lms_lti_connection=self.lti_provider
+        )
+
+        self.test_cg = CollectionGroup.objects.create(
+            name='TestColGroup',
+            owner=self.user,
+            engine=self.engine,
+            grading_policy=self.grading_policy
+        )
+        self.test_cg.collections.add(self.collection)
+        self.sequence = Sequence.objects.create(
+            lti_user=self.lti_user,
+            collection=self.collection,
+            group=self.test_cg,
+            outcome_service=self.outcome_service
+        )
+
+    @mock.patch('module.policies.policy_points_earned.update_lms_grades')
```

Comment:
I would change the import of the `update_lms_grades ` function in the policies from:
`from bridge_lti.outcomes import update_lms_grades` to:
`from bridge_lti import outcomes` and use the function as `outcomes.update_lms_grades` so we could patch required function directly in the parent module and continue using `ddt` library to unpack tests data.

Reply:
discussed with @idegtiarov and decided to not change this part of code and follow django style of imports.